swap_meet/item.py

```diff
@@ -1,2 +1,22 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category='', condition=0, age=0):
+        self.category = category
+        self.condition = condition
+        self.age = age
+
+    def __str__(self):
+        return "Hello World!"
+        
+    def condition_description(self):
+        descriptions = { 
+            0: "I should be paying you to take it off my hands.",
+            1: "Just don't touch your face after handling.",
+            2: "No refunds.",
+            3: "Not too shabby.",
+            4: "Looking good!",
+            5: "Absolutely immaculate. Exquisitely rare find!"
+        }
+        
+        self.condition = int(self.condition)
```

Comment:
Great way of fitting the potentially fractional condition into a valid key.

We should also think about what might happen if there's a condition value out of the expected range. What would happen if the condition was above 5, or below 0? Is that behavior OK?